x-pack/plugins/monitoring/server/routes/api/v1/logstash/pipelines/cluster_pipelines.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getClusterStatus } from '../../../../../lib/logstash/get_cluster_status';
+import { handleError } from '../../../../../lib/errors';
+import { getPaginatedPipelines } from '../../../../../lib/logstash/get_paginated_pipelines';
+import { MonitoringCore, PipelineMetricKey } from '../../../../../types';
+import { createValidationFunction } from '../../../../../lib/create_route_validation_function';
+import {
+  postLogstashClusterPipelinesRequestParamsRT,
+  postLogstashClusterPipelinesRequestPayloadRT,
+} from '../../../../../../common/http_api/logstash';
+
+const throughputMetric = 'logstash_cluster_pipeline_throughput';
+const nodesCountMetric = 'logstash_cluster_pipeline_node_count';
```

Comment:
Previously, this was logstash_cluster_pipeline_node**s**_count. However this is not a valid type according to [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/server/types.ts#L211). So we need to either include `logstash_cluster_pipeline_nodes_count` to the union or use a valid option here.

Reply:
I actually had to add `logstash_cluster_pipeline_nodes_count` to `PipelineNodeCountMetricKey`. The endpoint was returning 500 when `logstash_cluster_pipeline_node_count` was passed